client-app/app/models/message-collection.js

```diff
@@ -126,43 +198,47 @@ export default Em.Object.extend({
   },
 
   loadMore() {
-    const messages = this.get("messages");
-    if (messages.length === 0) {
+    const rows = this.rows;
+    if (rows.length === 0) {
       this.load({});
       return;
     }
 
-    const lastKey = messages[messages.length - 1].get("key");
+    const lastLog = rows[rows.length - 1];
+    const lastKey = lastLog.group ? lastLog.row_id : lastLog.key;
     this.load({
       after: lastKey
     });
   },
 
   hideCountInLoadMore: computed("search", "filter", function() {
-    const search = this.get("search");
-    const filter = this.get("filter");
+    const search = this.search;
```

Comment:
this is minor nitpick but I don't feel like we should have an intermediate const for this kind of cases, totally fine to have this.search in code